java/drivers/driver-hazelcast4plus/src/main/java/com/hazelcast/simulator/tests/cp/helpers/CpMapOperationCounter.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) 2008-2023, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hazelcast.simulator.tests.cp.helpers;
+
+import com.hazelcast.nio.ObjectDataInput;
+import com.hazelcast.nio.ObjectDataOutput;
+import com.hazelcast.nio.serialization.DataSerializable;
+
+import java.io.IOException;
+import java.util.concurrent.atomic.AtomicLong;
+
+public class CpMapOperationCounter implements DataSerializable {
+
+    public AtomicLong putCount = new AtomicLong(0);
```

Comment:
I would get rid of all the atomics since this class isn't shared between threads. No need to burden the CPU with atomic instructions.

Reply:
done